test/integrations/validation/Zod.spec.ts

```diff
@@ -0,0 +1,348 @@
+
+import { expect, describe, it } from 'vitest';
+
+import zod from '../../../src/integrations/validation/module';
+
+import { SCHEMAS, ERRORS } from './_fixtures/Zod.fixtures';
+
+describe('Zod', () =>
```

Comment:
We use Zod as implementation, but we've created this test to test other implementations at the moment we need to switch. Therefore this test isn't Zod specific. I want to consider it as a generic validator test.